Makefile

```diff
@@ -177,6 +187,33 @@ vendor: $(GO)
 tools: $(GO)
 	./hack/install-tools.sh
 
+# Generate bundle manifests and metadata, then validate generated files.
+bundle: gen-crds manifests
```

Comment:
The upgrade guide for 1.2.0 says this should use kustomize: https://github.com/operator-framework/operator-sdk/blob/master/website/content/en/docs/upgrading-sdk-version/v1.2.0.md#golang-based-operators-update-makefiles-bundle-target

Reply:
That is planned. Just want to get to this sooner than later.